server/src/com/cloud/network/router/VirtualNetworkApplianceManagerImpl.java

```diff
@@ -2627,7 +2627,7 @@ public boolean postStateTransitionEvent(final StateMachine2.Transition<State, Vi
                 s_logger.info("Schedule a router reboot task as router " + vo.getId() + " is powered-on out-of-band. we need to reboot to refresh network rules");
```

Comment:
@DaanHoogland I see that you introduced RouterReprovisionOnOutOfBandMigration as part of commit b9dd67c383874ef7273205e0eac831de497bb02d. Wasn't this config just meant to decide between VR reboot and generating alert? But I see that both 'if' and 'else' conditions are different. Has something changed?